pkg/api/runner.go

```diff
@@ -12,6 +13,7 @@ type Runner struct {
 	BufferSize            int
 	MaxConcurrentRequests int
 	ConcurrentRecorders   int
+	RequestTimeout        time.Duration
```

Comment:
Alternatively you could define as a pointer `*time.Duration`. This has the advantage that it is less ambiguous around whether a value is needed or not. Your current approach works fine, it's just odd to see that the default in command line is `0s` (users may get confused as to what that even means).